planet/exceptions.py

```diff
@@ -13,8 +13,13 @@
 # limitations under the License.
 
 
-class APIException(Exception):
-    '''General unexpected response'''
+class PlanetException(Exception):
```

Comment:
@sgillies just want to point out that I added this in here. Nice to have a root exception for the project instead of just extending `Exception` directly.